src/main/java/edu/cornell/kfs/ksr/businessobject/SecurityProvisioningGroup.java

```diff
@@ -0,0 +1,202 @@
+package edu.cornell.kfs.ksr.businessobject;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.kuali.kfs.core.api.mo.common.active.Inactivatable;
+import org.kuali.kfs.kim.impl.role.Role;
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectBase;
+
+public class SecurityProvisioningGroup extends PersistableBusinessObjectBase implements Inactivatable, KsrObjectWithRoles {

```

Comment:
Although the Cynergy KSR code implemented the Inactivatable interface, it might make more sense to implement MutableInactivatable instead (and add Override annotations as needed), given that its active flag is mutable.

Reply:
updated